observability/monitoring/managing-alerts/managing-alerts-as-a-developer.adoc

```diff
@@ -0,0 +1,68 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="managing-alerts-as-a-developer"]
+= Managing alerts as a Developer
+:context: managing-alerts-fas-a-developer
+
+toc::[]
+
+In {product-title}, the Alerting UI enables you to manage alerts, silences, and alerting rules.
+
+[NOTE]
+====
+The alerts, silences, and alerting rules that are available in the Alerting UI relate to the projects that you have access to.
+====
+
+// Accessing the Alerting UI from the Developer perspective
+include::modules/monitoring-accessing-the-alerting-ui.adoc[leveloffset=1,tags=**;DEV;!ADM]
+
+// Getting information about alerts, silences and alerting rules from the Developer perspective
+include::modules/monitoring-getting-information-about-alerts-silences-and-alerting-rules.adoc[leveloffset=1,tags=**;DEV;!ADM]
+
+[role="_additional-resources"]
+.Additional resources
+* See the link:https://github.com/openshift/runbooks/tree/master/alerts/cluster-monitoring-operator[{cmo-full} runbooks] ({cmo-full} GitHub repository) to help diagnose and resolve issues that trigger specific {product-title} monitoring alerts.
```

Comment:
Per "Additional resources" [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections), should just be a link w/ no surrounding text / sentence:
```suggestion
* link:https://github.com/openshift/runbooks/tree/master/alerts/cluster-monitoring-operator[{cmo-full} runbooks] ({cmo-full} GitHub repository)
```